File: Return_Negative.c
#include<stdio.h>
#include<stdlib.h>

int makeNegative(int num)
{
	if (num > 0)
		return (num * -1);
	else
		return (num);
}

int make_Negative(int num)
{
	return (num > 0 ? (num * -1) : num);
}

int main()
{
	// Test cases
	int positiveNumber = 5;
	int negativeNumber = -8;
	int zero = 0;

	// Test makeNegative function
	printf("makeNegative:\n");
	printf("%d becomes %d\n", positiveNumber, makeNegative(positiveNumber));
	printf("%d remains %d\n", negativeNumber, makeNegative(negativeNumber));
	printf("Zero remains %d\n", makeNegative(zero));

	// Test make_Negative function
	printf("\nmake_Negative:\n");
	printf("%d becomes %d\n", positiveNumber, make_Negative(positiveNumber));
	printf("%d remains %d\n", negativeNumber, make_Negative(negativeNumber));
	printf("Zero remains %d\n", make_Negative(zero));

	return (0);
}

File: Return_Negative.py
def makeNegative(num):
    return (num * -1) if (num > 0) else num

positive_number = 5
negative_number = -8
zero = 0

# Test makeNegative function
result_positive = makeNegative(positive_number)
result_negative = makeNegative(negative_number)
result_zero = makeNegative(zero)

# Print results
print(f"{positive_number} becomes {result_positive}")
print(f"{negative_number} remains {result_negative}")
print(f"{zero} remains {result_zero}")